frontend/next.config.js

```diff
@@ -76,6 +76,27 @@ const nextConfig = {
           },
         ],
       },
+      // don't cache user specific pages: saved-grants, saved-search-queries
+      {
+        source: "/saved:path*",
```

Comment:
Since the `/saved-*` paths are all behind pages that require a session cookie, I think this entry might not be needed. Testing locally I was able to see the refreshes happen as they are supposed to with just the cookie entry.

Reply:
yeah, we might want to try running it rebased off of this see how things work in that world https://github.com/HHS/simpler-grants-gov/pull/4455

Reply:
without the redirect, removing the /saved* declaration things work for me locally on a local prod build, so I think we can remove that declaration?

Reply:
When clicking with the link component from an opportunity the `saved-grants` page is loaded through client side navigation, which in the browser is a fetch, ie `http://localhost:3000/saved-grants?_rsc=hp2fn`. That fetch was getting cached before which the cookie entry fixes.

I was able to recreate an experience where you login in tab 1, logout tab 2, click saved grants in tab 1, login in tab 2, and click saved grants it does initially load the cached page, so let's leave it in as is.